pages/_app.tsx

```diff
@@ -13,6 +14,7 @@ function App({ Component, pageProps }: AppProps): ReactElement {
 
   const dispatch = useAppDispatch();
   useEffect(() => {
+    TagManager.initialize({ gtmId: process.env.NEXT_PUBLIC_GTM_ID! });
```

Comment:
Something seems odd about this approach. I've never used the library before, but you usually have to embed GTM in the page head. If it's going in a useEffect then I'm guessing it's not loading GTM until React is bootstrapped and that's going to cause inaccurate page load time values in Google Analytics as well as make adding tags to the page load event difficult.

Reply:
Details here: https://www.npmjs.com/package/react-gtm-module
I actually noticed this was used elsewhere in a Hackney repo, but yeah alternative I guess would be to add to the `_document.tsx`